Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -200,5 +203,35 @@ private BocList GetControlToValidate ()
 
       return bocListControl;
     }
+
+    void ILazyValidator.LazyValidate ()
+    {
+      var bocList = GetControlToValidate();
+      var validationRepository = bocList.ValidationFailureRepository;
+
+      var sb = new StringBuilder();
+
+      foreach (var validationFailure in _validationFailuresForBusinessObjectPropertyOfBocList)
+      {
+        sb.AppendLine(validationFailure.Failure.ErrorMessage);
+      }
+
+      var rowObjects = bocList.Value ?? Array.Empty<IBusinessObject>();
+
+      var hasRowsWithValidationFailures = rowObjects.Any(r => validationRepository.HasUnhandledValidationFailuresForDataRow(r));
```

Comment:
If we move the evaluation of the error message to here, we can use the "get me all unhandled validation failures" method instead.